quickwit-storage/src/cache/quickwit_cache.rs

```diff
@@ -166,6 +175,18 @@ mod tests {
         quickwit_cache.get_all(Path::new("bubu/toto.fast")).await;
     }
 
+    #[test]
+    fn test_fast_cache_capacity_from_env() {
+        assert_eq!(
+            super::fast_cache_capacity(),
+            super::DEFAULT_FAST_CACHE_CAPACITY
+        );
+        std::env::set_var(super::FAST_CACHE_CAPACITY_ENV_KEY, "3000000001");
+        assert_eq!(super::fast_cache_capacity(), 3_000_000_001);
+        std::env::set_var(super::FAST_CACHE_CAPACITY_ENV_KEY, "invalidnumber");
+        assert_eq!(super::fast_cache_capacity(), 3_000_000_000);
```

Comment:
we should print at least a error here or the user will not know the value is rejected

Reply:
Great point